common-custom-user-data-gradle-plugin/README.md

```diff
@@ -2,17 +2,39 @@
+Out of the box, this plugin will enhance all Build Scans published with additional tags, links and custom values. These include:
+- A tag representing the Operating System.
+- A tag representing how the build was invoked, be that from your IDE (IDEA, Eclipse, Android Studio) or from the command-line.
+- A `CI` tag for builds run on CI. In addition, a set of tags, links and custom values specific to the CI server will be added.
+  - The following CI servers are supported: Jenkins, Team City, Circle CI, Bamboo, GitHub Actions, Gitlab and Travis.
+- For `git` repositories, custom values with the Commit ID, Branch and the output of `git status`
 
-If you are still using a 5.x version of Gradle, apply the Common Custom User Gradle plugin to the `build.gradle[.kts]` file of your root project.
+### Applying the published plugin
+
+The Common Custom User Data Gradle Plugin is available in the [Gradle Plugin Portal](https://plugins.gradle.org/plugin/com.gradle.common-custom-user-data-gradle-plugin). This plugin requires the the [Gradle Enterprise Gradle plugin](https://plugins.gradle.org/plugin/com.gradle.enterprise) also be applied in your build in order to have an effect.
+
+To benefit from the Common Custom User Gradle plugin in your project, you must apply it in the `settings.gradle[.kts]` file of your project. This is the same file where you have already declared the Gradle Enterprise Gradle plugin. See [here](https://github.com/gradle/gradle-enterprise-build-config-samples/blob/master/common-custom-user-data-gradle-plugin/settings.gradle) for an example.
+
+If you are still using a 5.x version of Gradle, apply the plugin to the `build.gradle[.kts]` file of your root project.
+
+### Developing a customized version of the plugin
+
+For more power and flexibility, we recommend creating a copy of this repository so that you may develop a customized version of the plugin and publish it locally for your projects to consume.
+
+This approach has a number of benefits:
+- Add, remove and change the set of tags, links and custom values added to each Build Scan. This allows you to customize your Build Scans for your particular needs.
+- Standardize Gradle Enterprise and Gradle Build Cache configuration in your organization, removing the need for each project to specify this configuration.
+
+This approach can be particularly useful when Gradle Enterprise and Build Cache configuration should be shared among a number of Gradle projects. If your customized plugin provides all required Gradle Enterprise configuration, then each consumer project will get all the benefits of Gradle Enterprise simply by applying this single plugin. The plugin sources provide a placeholder and example code to get you started.
```

Comment:
The first sentence seems to be a repetition of the second item in the bullet list right above. I think either remove the duplication or somehow connect them better. In its current form, I stumbled over this when reading.